src/content/docs/en/guides/typescript.mdx

```diff
@@ -67,6 +67,20 @@ Then, add the following to your `tsconfig.json`:
 
 To check that the plugin is working, create a `.ts` file and import an Astro component into it. You should have no warning messages from your editor.
 
+### Types in your Astro config file
+
+You may want type hints and errors in your `astro.config.*` file as well. When using the `strict` or `strictest` TypeScript configurations, you can use the `.ts` extension instead of the `.mjs` extension to enable TypeScript parsing. The `.ts` extension is also necessary to support generated types from Astro integrations, [including Astro DB](/en/guides/astro-db/).
```

Comment:
Debating a change from "you can use" to "we _recommend_ using." That would be a strong stance since we DON'T recommend `.ts` today. But perhaps we should!

Reply:
So in general, we would remove all tone of that sort and just say something like:

"For type hints and errors in your `astro.config` file..." and just say what it requires to get that. We don't have to make any judgments, recommendations etc. Just "To get X, do Y."

Note that we also avoid "you CAN" when we mean "you MUST do Y to get X". That wording is almost exclusively reserved for a choice: "You CAN set a default, but you don't have to."